hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,111 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+
+  build_opts: ["-sv",
+               "-timescale 1ns/1ps",
+               "+incdir+{QUESTA_HOME}/verilog_src/uvm-1.2/src/",
+               "{QUESTA_HOME}/verilog_src/uvm-1.2/src/uvm_pkg.sv",
+               // Multi-file compilation unit option for Questa
+               // Needed to use macros across CU-s
+               "-mfcu"
+               "-f {sv_flist}"]
+
+  run_opts:   ["-sv_seed {seed}",
+               // List multiple tops for the simulation
+               "{sim_tops}",
+               // dv_macros.svh has a macro printing null using %0d
+               // format specifier, Questa throws an error on this 
+               // we demote this error in Questa
+               "-suppress vsim-8323",
+               "-c",
+               "-sv_lib {QUESTA_HOME}/uvm-1.2/linux_x86_64/uvm_dpi",
+               "-lib {sv_flist_gen_dir}/work",
+	       // Questa forces all declared virtual interfaces to be allocated, 
```

Comment:
nit: The alignment is slightly off.

Reply:
This PR is ready to go in, once you fix this.

Reply:
@sriyerg I did fix it locally in my repo. Should I rebase and submit again? Is it OK to do it in next PR please? I am moving across machines (MAC/Linux) and is harder to track it (mess at my end).

Reply:
Please rebase and resubmit again if possible. Thanks!

Reply:
I followed #4633 and did:

```log
git rebase -i upstream/master
git commit --amend -s 
git push -f
```

But I don't quite see the extra space character in this PR  yet, maybe I am not seeing the right version in the PR. Can you please confirm if I am missing any steps in this update?

Thanks